internal/controller/controllers/crd_events_handler.go

```diff
@@ -0,0 +1,53 @@
+package controllers
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+)
+
+//go:generate mockgen -package controllers -destination mock_crd_events_handler.go . CRDEventsHandler
+type CRDEventsHandler interface {
+	NotifyUpdates(ch chan<- event.GenericEvent, name string, namespace string)
```

Comment:
if it's used only internal then it doesn't belong to the interface.

Reply:
Done